app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -462,6 +558,7 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope {
     }
 
     fun refresh() {
+        swipeContainer.isRefreshing = true
```

Comment:
>Do not call this when refresh is triggered by a swipe gesture.

The documentation on `SwipeRefreshLayout` suggests we shouldn't set this here?

Reply:
This is triggered when the button in the Popup Menu is pressed, so it's necessary.